lua/ts_utils.lua

```diff
@@ -105,7 +105,7 @@ end
 
 function M.has_parser(lang)
   local lang = lang or api.nvim_buf_get_option(0, 'filetype')
-  return #api.nvim_get_runtime_file('parser/' .. lang .. '.*', false) > 0
+  return #lang > 0 and #api.nvim_get_runtime_file('parser/' .. lang .. '.*', false) > 0
```

Comment:
parsers should end with .so though, shouldn't this just check for `.so` like nvim-treesitter does ? It would fix everything.

Reply:
This will be fixed in a future update, as we are going to move to using `nvim-treesitter` as a base for the plugin